proto/internal/core/v1/core.proto

```diff
@@ -211,4 +219,10 @@ message ComputedUserset {
     pattern : "^[a-z][a-z0-9_]{1,62}[a-z0-9]$",
     max_bytes : 64,
   } ];
-}
\ No newline at end of file
+  SourcePosition source_position = 3;
+}
+
+message SourcePosition {
+  uint64 zero_indexed_line_number = 1;
```

Comment:
Why are these zero based but the struct and errors in common 1 based?

Reply:
Because I prefer 0-indexed in general, but we used 1-indexed in developer API because proto doesn't have optional fields. Here, however, it is in a message which itself is optional, so I figured I could use 0 like I wanted